backend/capellacollab/settings/modelsources/t4c/repositories/crud.py

```diff
@@ -85,6 +85,7 @@ def _get_user_write_t4c_repositories(
         .join(toolmodels_models.DatabaseCapellaModel.version)
         .where(tools_models.DatabaseVersion.name == version_name)
         .join(toolmodels_models.DatabaseCapellaModel.project)
+        .where(projects_models.DatabaseProject.is_archived.is_(False))
```

Comment:
```suggestion
        .where(projects_models.DatabaseProject.is_archived == False)
```
Why don't you use this syntax?

Reply:
Ah, this is annoying.